src/ansys/dpf/composites/layup_info/material_operators.py

```diff
@@ -145,12 +145,15 @@ def get_material_operators(
     # Combines the material support in the engineering data XML file and the unit system.
     # Its output can be used to evaluate material properties.
     material_provider = Operator("eng_data::ans_mat_material_provider")
-    material_provider.inputs.data_sources = engineering_data_source
```

Comment:
Unknown why this worked at all but the DataSource for the ENGD file is set on line 152.